### ISSUE TITLE
Next release?

### ISSUE THREAD

**Issue by nickpeihl** [Esri/esri-leaflet-renderers#140]
Hi @jgravois 
It's been a while since this repo has had a new release. Is there anything blocking it? Happy to help if I can.

**Comment by jgravois** [comment]
hi @nickpeihl!

> Is there anything blocking it?

nothing in the codebase. i was just having trouble running the tests locally and too lazy to figure out how to resolve the problem.

thanks for the kick in the pants.  [`v2.0.5`](https://github.com/Esri/esri-leaflet-renderers/releases/tag/v2.0.5) is now out the 🚪 .

**Comment by nickpeihl** [comment]
My hero! 😍

**Comment by jgravois** [comment]
not so fast. my hasty attempt to PEMDAS lint fix for semistandard 10.0 [here](https://github.com/Esri/esri-leaflet-renderers/commit/7851c3f96814f41e39969a3dc6488840ff10ab38#diff-5c1b542c7b0a9148c576c44c6584bb89L124) introduced a bug.

```js
// is
interpolatedColor[j] = (Math.round(lowerBoundColor[j] * lowerBoundColorWeight) + (upperBoundColor[j] * upperBoundColorWeight));

// should be 
interpolatedColor[j] = Math.round((lowerBoundColor[j] * lowerBoundColorWeight) + (upperBoundColor[j] * upperBoundColorWeight));
```

ugh. `2.0.7` will follow.
